tests/root_test.py

```diff
@@ -10,19 +12,13 @@ def test_root(stapi_client: TestClient, url_for) -> None:
     assert res.status_code == status.HTTP_200_OK
     assert res.headers["Content-Type"] == "application/json"
 
-    data = res.json()
-
-    link = find_link(data["links"], "self")
-    assert link, "GET / Link[rel=self] should exist"
-    assert link["type"] == "application/json"
-    assert link["href"] == url_for("/")
+    body = res.json()
 
-    link = find_link(data["links"], "service-description")
-    assert link, "GET / Link[rel=service-description] should exist"
-    assert link["type"] == "application/json"
-    assert str(link["href"]) == url_for("/openapi.json")
+    assert body["conformsTo"] == [CORE]
 
-    link = find_link(data["links"], "service-docs")
-    assert link, "GET / Link[rel=service-docs] should exist"
-    assert link["type"] == "text/html"
-    assert str(link["href"]) == url_for("/docs")
+    assert_link("GET /", body, "self", "/", url_for)
+    assert_link("GET /", body, "service-description", "/openapi.json", url_for)
+    assert_link("GET /", body, "service-docs", "/docs", url_for, media_type="text/html")
+    assert_link("GET /", body, "conformance", "/conformance", url_for)
+    assert_link("GET /", body, "products", "/products", url_for)
+    assert_link("GET /", body, "orders", "/orders", url_for)
```

Comment:
I wonder if for readability using like this would be easier? Same amount of characters, but _I_ think the intent of both "orders" and url_for is more evident?
 
```suggestion
    assert_link("GET /", body, "orders", url_for("/orders"))
```